src/main/java/io/appium/java_client/ios/TVOSDriver.java

```diff
@@ -0,0 +1,176 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.appium.java_client.ios;
+
+import static org.openqa.selenium.remote.CapabilityType.PLATFORM_NAME;
+
+import io.appium.java_client.AppiumDriver;
+import io.appium.java_client.FindsByIosClassChain;
+import io.appium.java_client.FindsByIosNSPredicate;
+import io.appium.java_client.HasOnScreenKeyboard;
+import io.appium.java_client.HidesKeyboardWithKeyName;
+import io.appium.java_client.LocksDevice;
+import io.appium.java_client.remote.MobilePlatform;
+import io.appium.java_client.screenrecording.CanRecordScreen;
+import io.appium.java_client.service.local.AppiumDriverLocalService;
+import io.appium.java_client.service.local.AppiumServiceBuilder;
+import io.appium.java_client.ws.StringWebSocketClient;
+import org.openqa.selenium.Capabilities;
+import org.openqa.selenium.MutableCapabilities;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.remote.HttpCommandExecutor;
+import org.openqa.selenium.remote.http.HttpClient;
+
+import java.net.URL;
+import javax.annotation.Nullable;
+
+/**
+ * Apple tvOS driver implementation.
+ *
+ * @param <T> the required type of class which implement
+ *           {@link WebElement}.
+ *           Instances of the defined type will be returned via findElement* and findElements*.
+ *           Warning (!!!). Allowed types:
+ *           {@link WebElement}
+ *           {@link org.openqa.selenium.remote.RemoteWebElement}
+ *           {@link io.appium.java_client.MobileElement}
+ *           {@link IOSElement}
+ */
+public class TVOSDriver<T extends WebElement>
+    extends AppiumDriver<T>
+    implements HidesKeyboardWithKeyName, HasIOSSettings, HasOnScreenKeyboard,
+        LocksDevice, FindsByIosNSPredicate<T>, FindsByIosClassChain<T>,
+        PushesFiles, CanRecordScreen, HasIOSClipboard {
```

Comment:
Did you check all of commands work? (just a confirmation.)
Clipboad does not work on this list.